push/push.go

```diff
@@ -109,9 +109,9 @@ func GithubPush(ctx context.Context, input Input, repoLimiter *time.Ticker, push
 	// Title is first line of commit message.
 	// Body is given by body-file if it exists or is the remainder of the commit message after title.
 	title := input.CommitMessage
-	body := ""
+	body := input.PRBody
 	splitMsg := strings.SplitN(input.CommitMessage, "\n", 2)
-	if len(splitMsg) == 2 {
+	if body == "" && len(splitMsg) == 2 {
```

Comment:
```suggestion
	if len(splitMsg) == 2 {
```
I think this should remain unchanged, since otherwise the title of the PR would be multiple lines

Reply:
Hm. But on the other hand, in that case whatever comes after the line break in `CommitMessage` would be discarded if a body was given? I'm not sure what I'd expect in the interaction between giving a multi-line commit message *and* setting a body file.
If that is the desired behaviour, then I agree with your suggestion!

Reply:
Yeah, perhaps in that case the body should be a combination of what comes after the line break in `CommitMessage` and PR body